src/pages/derivx/_accounts.tsx

```diff
@@ -18,7 +23,7 @@ const StyledIcon = styled.img`
     }
 `
 
-const content = [
+const content: { header: JSX.Element; text: JSX.Element | string; icon: JSX.Element | string }[] = [
```

Comment:
@NikitaK-deriv  can you declare this as a new type instead of putting it inline, also can you change `JSX.Element` to `ReactElement` instead?